optaplanner-examples/src/main/java/org/optaplanner/examples/meetingscheduling/domain/Meeting.java

```diff
@@ -86,4 +105,11 @@ public String toString() {
         return topic;
     }
 
+    public boolean isEntireGroupMeeting() {
```

Comment:
Why have this getter/setter combo away from the others.
Keep the simple getters and setters all together in the file, in the order in which their fields are declared.
That way it's easier to review the less predictable methods, such as toString() and getDurationString() when doing code changes.